go.mod

```
@@ -1,6 +1,6 @@
 module github.com/prometheus/alertmanager
 
-go 1.18
+go 1.23.0
```

this might be actually broken?

```suggestion
go 1.23
```
https://docs.google.com/document/d/14RT5Ns7c1BWOU6FwFjPj0N1TGA_mjw7y4VTaevALx8g/edit?tab=t.0

---

I think that's fine. `go mod tidy` creates that, and it builds in Docker no problem. I'll ignore for now, but if it breaks for some reason, I'll come back to this.